### PR TITLE
Update JAX installation guide

### PR DESCRIPTION
**Pull request by francois-rozet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,18 +67,24 @@ Host alan
 ```
 to `~/.ssh/config`.
 
-### Preparing an Anaconda environment
+### Preparing an conda environment
```
Comment (glouppe) [inline README.md:70]: an -> a

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Table of contents:
 - [General actions](#general-actions)
 - [User account setup](#user-account-setup)
   - [Connecting to Alan](#connecting-to-alan)
-  - [Preparing an Anaconda environment](#preparing-an-anaconda-environment)
+  - [Preparing an conda environment](#preparing-an-conda-environment)
```
Comment (glouppe) [inline README.md:20]: an -> a

**Review by glouppe — COMMENTED** [review]
I tried installing and executing Jax in a new environment:

```
(base) [glouppe@master ~]$ conda create -n jax-test
[...]

(base) [glouppe@master ~]$ conda activate jax-test
(jax-test) [glouppe@master ~]$ pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
[...]

(jax-test) [glouppe@compute-05 ~]$ python
Python 3.11.4 (main, Jul  5 2023, 13:45:01) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import jax
>>> jax.default_backend()
'gpu'
>>> jax.devices()
[gpu(id=0)]
>>> import jax.numpy as jnp
>>> from jax import grad, jit, vmap
>>> from jax import random
>>> key = random.PRNGKey(0)
Could not load library libcudnn_ops_infer.so.8. Error: libcublas.so.11: cannot open shared object file: No such file or directory
```

**Comment by francois-rozet** [comment]
It works for me with `python=3.9`.

```console
(jaxtest) [frozet@compute-02 ~]$ python
Python 3.9.16 | packaged by conda-forge | (main, Feb  1 2023, 21:39:03) 
[GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import jax
>>> jax.jit(jax.vmap(jax.grad(lambda x: x ** 2)))(jax.numpy.linspace(0, 1, 5))
Array([0. , 0.5, 1. , 1.5, 2. ], dtype=float32)
>>> jax.devices()
[gpu(id=0)]
```

**Pull request merged**